app/views/includes/foot.jade

```diff
@@ -44,5 +45,22 @@ script(type='text/javascript', src='/js/init.js')
 //Socket.io Client Library
 script(type='text/javascript', src='/socket.io/socket.io.js')
 
+// firebase
+script(type='text/javascript', src='https://www.gstatic.com/firebasejs/3.5.0/firebase.js')
+script(type='text/javascript', src='https://www.gstatic.com/firebasejs/3.5.0/firebase-app.js')
+script(type='text/javascript', src='https://www.gstatic.com/firebasejs/3.5.0/firebase-database.js')
+script(type='text/javascript', src='https://www.gstatic.com/firebasejs/3.5.0/firebase-messaging.js')
+script(type='text/javascript', defer src="https://code.getmdl.io/1.1.3/material.min.js")
```

Comment:
I will go ahead and remove the mdl script tags since we are using materialize now.